shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -234,6 +234,16 @@ - (void)setupNotificationCenterObservers {
                  name:UIAccessibilityBoldTextStatusDidChangeNotification
                object:nil];
 
+  [center addObserver:self
+             selector:@selector(onAccessibilityStatusChanged:)
+                 name:UIAccessibilityOnOffSwitchLabelsDidChangeNotification
+               object:nil];
+
+  [center addObserver:self
+             selector:@selector(onAccessibilityStatusChanged:)
+                 name:UIAccessibilityOnOffSwitchLabelsDidChangeNotification
+               object:nil];
```

Comment:
When this is re-landed, only one of these two duplicate notifications should be applied.